environments/gcp/flyte-core/README.md

```diff
@@ -52,7 +52,7 @@ backend "gcs" {
 ```bash
 terraform init
 ```
-9. Then:
+9. Then change to the environements/gcp/flyte-core directory and run:
```

Comment:
```suggestion
9. Then change to the environments/gcp/flyte-core directory and run:
```
We can also make this a step at the beginning.

Meta question: do you think moving to using tfvars would be better than asking users to edit files?